recipes/qt/5.x.x/conandata.yml

```diff
@@ -0,0 +1,14 @@
+sources:
+  "5.15.2":
+    url:
+      - "https://download.qt.io/archive/qt/5.15/5.15.2/single/qt-everywhere-src-5.15.2.tar.xz"
+      - "https://qt-mirror.dannhauer.de/archive/qt/5.15/5.15.2/single/qt-everywhere-src-5.15.2.tar.xz"
+      - "https://www.funet.fi/pub/mirrors/download.qt-project.org/archive/qt/5.15/5.15.2/single/qt-everywhere-src-5.15.2.tar.xz"
+      - "https://ftp.fau.de/qtproject/archive/qt/5.15/5.15.2/single/qt-everywhere-src-5.15.2.tar.xz"
+    sha256: "3a530d1b243b5dec00bc54937455471aaa3e56849d2593edb8ded07228202240"
+patches:
+  "5.15.2":
+    - patch_file: "patches/aa2a39dea5.diff"
```

Comment:
It looks like you are only appling 2 out of 4 patches.

Reply:
Thanks for noticing, it's because the PR started with another version of qt. I'll remove unused files.